packages/dds/tree/src/simple-tree/api/schemaFactory.ts

```diff
@@ -134,17 +134,28 @@ export type ScopedSchemaName<
- * | Allows adding "local" (non-persisted) members | Yes. Members (including methods) can be added to class.        | No. Attempting to set non-field members will error. |
- * | Prototype | The user defined class | `Object.prototype`, `Map.prototype` or `Array.prototype` depending on node kind |
- * | Structurally named Schema | Not Supported | Supported |
- * | Explicitly named Objects | Supported | Supported |
- * | Explicitly named Maps and Arrays | Supported: Both declaration approaches can be used | Not Supported |
- * | node.js assert.deepEqual | Compares like class instances: equal to other nodes of the same type with the same content, including custom local fields. | Compares like plain objects: equal to plain JavaScript objects with the same fields, and other nodes with the same fields, even if the types are different. |
- * | IntelliSense | Shows and links to user defined class by name: `X` | Shows internal type generation logic: `object & TreeNode & ObjectFromSchemaRecord<{}> & WithType<"test.x">` |
- * | Recursion | Supported with special declaration patterns. | Unsupported: Generated d.ts files replace recursive references with `any`, breaking use of recursive schema across compilation boundaries |
+ *
+ * Customizable Approach
+ * 1. Declaration: `class X extends schemaFactory.object("x", {}) {}`
+ * 2. Allows adding "local" (non-persisted) members: Yes. Members (including methods) can be added to the class.
+ * 3. Prototype: The user-defined class.
+ * 4. Structurally named Schema: Not Supported.
+ * 5. Explicitly named Objects: Supported.
+ * 6. Explicitly named Maps and Arrays: Supported: Both declaration approaches can be used.
+ * 7. Node.js `assert.deepEqual`: Compares like class instances: equal to other nodes of the same type with the same content, including custom local fields.
+ * 8. IntelliSense: Shows and links to user-defined class by name: `X`.
+ * 9. Recursion: Supported with special declaration patterns.
+ *
+ * POJO Emulation Approach
+ * 1. Declaration: `const X = schemaFactory.object("x", {}); type X = NodeFromSchema<typeof X>;`
+ * 2. Allows adding "local" (non-persisted) members: No. Attempting to set non-field members will result in an error.
+ * 3. Prototype: `Object.prototype`, `Map.prototype`, or `Array.prototype` depending on node kind.
+ * 4. Structurally named Schema: Supported.
+ * 5. Explicitly named Objects: Supported.
+ * 6. Explicitly named Maps and Arrays: Not Supported.
+ * 7. Node.js `assert.deepEqual`: Compares like plain objects: equal to plain JavaScript objects with the same fields, and other nodes with the same fields, even if the types are different.
+ * 8. IntelliSense: Shows internal type generation logic: `object & TreeNode & ObjectFromSchemaRecord<{}> & WithType<"test.x">`.
+ * 9. Recursion: Unsupported: Generated `.d.ts` files replace recursive references with `any`, breaking the use of recursive schema across compilation boundaries.
```

Comment:
Would probably be good to drop a comment in the privateRemarks noting why this isn't using a table.